error-analyzer/src/main/java/com/google/sps/servlets/helloInLog4j2.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+@WebServlet("/")
+public class helloInLog4j2 extends HttpServlet {
+
+    private static final Logger logger = LogManager.getLogger(helloInLog4j2.class);
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        System.out.println("------------------------------------------------");
+        logger.debug("This is a debug message");
```

Comment:
There's "trace" as well right?

Reply:
Added a trace statement now. Since the level of debug is set to debug, the trace log will not appear in log file